docker/dockerfiles/base

```diff
@@ -21,7 +21,7 @@ RUN apt-get update && apt-get install apt-transport-https && \
             libmariadbclient-dev mariadb-client && \
     rm -rf /var/lib/apt/lists/*
 
-COPY ./requirements/ /app/requirements/
+COPY ./requirements/*.txt /app/requirements/
```

Comment:
Is this what we need to get proper requirements caching?

Reply:
I'm hoping so. It's kindof a guess as the docs aren't clear on this, but I think it could help since it's the documented way of copying multiple files in a dir.